tests/test-attachments.md

```diff
@@ -68,6 +68,10 @@ Expected: Generic Powerpoint thumbnail & preview window.
 Expected: Generic Word thumbnail & preview window.  
 [Permalink](https://pre-release.mattermost.com/core/pl/d7s76wp6kjdwfp4hgrwpijsnyy)
 
+**Text**  
+`Documents/Text.txt`  
+Expected: Generic Text thumbnail & file previewer.
```

Comment:
It seems to show up on master as a code thumbnail (see screenshot). If this is expected then propose we just change to `generic code thumbnail & file previewer`. Also is there a max length for the file previewer? eg PDF previews only show first 5 pages.

![image](https://cloud.githubusercontent.com/assets/14333569/19940480/6f478516-a0f2-11e6-84cd-33112a4d031d.png)